doc/manual/install-language.rst

```diff
@@ -0,0 +1,90 @@
+Appendix: Installing the example languages
+==========================================
+
+DOMjudge default ships with some default languages with a default configuration.
+As you might setup contests with those languages we provide how those languages were
+installed in the past as guideline.
+
+Most of those can be found in the table below as there is a package available to install
+inside the judging chroot. Given that you can install your own chroot we only provide the
```

Comment:
Perhaps mention how it can be installed in the chroot?

Reply:
Yes, will do.